tests/e2e/sidecar/sidecar-pod/00-install.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
```

Comment:
Use the `render_install_jaeger` function from render.sh to generate this file and the assert file.